EpsilonAuxBMS/AuxBMSTests.adoc

```diff
@@ -0,0 +1,102 @@
+= Aux BMS Tests
+
+== HeartBeat Test
+----
+Elapsed 5000ms{
+	Expect 5 Heartbeat CAN messages (0x720)
+}
+----
+
+== Aux Test
+----
+Elapsed 1000ms{
+	Expect 10 Aux CAN messages (0x721)
+}
+----
+
+== CAN Values to Test
+[options="header"]
+|===
+| Input Type | CAN_ADDRESS | DATA_INDEX | TIME_FRAME
+| PrechargeState |  0x721  | 0 |  100ms 
+| AuxVoltage |  0x721 |  0 |  100ms |
+|===
+
+== Precharge Test
+----
+Send (PRECHARGE_CAN_MESSAGE(0x721)[DATA_INDEX])
+Within 100ms {
+    Expect (PRECHARGE_STATUS_CAN_MESSAGE(0x721)[DATA_INDEX])
+}
+----
+
+== AuxVoltage Test
+----
+Every 100ms{
+    Expect (CURRENT_VOLTAGE_AUX_BATTERY_CAN_MESSAGE(0x721)[DATA_INDEX])
+}
+----
+
+== Contactor Charge Test
+----
+Every 100ms{
+	Expect (CONTACTOR_CURRENT_CAN_MESSAGE(0x721)[DATA_INDEX])
+}
+----
+
+== Contactor State Test
+----
+Every 100ms{
+	Expect (CONTACTOR_STATE_CAN_MESSAGE(0x721)[DATA_INDEX])
+}
+----
+
+== All Contactor On Test
+----
+Charge input test
+
+Wait 5s{
```

Comment:
Wait isn't really a good term in a test. Better would be, `within` would be better. The main concern I have here is that there is no accepted range. Is it between 4 and 5 seconds?